src/i18n/strings/cs.json

```diff
@@ -3495,7 +3498,6 @@
             "sent": "%(senderName)s pozval(a) uživatele %(targetDisplayName)s ke vstupu do místnosti."
         },
         "m.room.tombstone": "%(senderDisplayName)s aktualizoval(a) místnost.",
-        "m.room.topic": "%(senderDisplayName)s změnil(a) téma na „%(topic)s“.",
```

Comment:
I did reimport these translations under the new key in LL, so not sure why a rerun of the Download action didn't handle it.